src/runtime.rs

```diff
@@ -63,6 +64,24 @@ pub fn cnb_runtime<P: Platform, BM: DeserializeOwned, E: Debug + Display>(
     if let Err(lib_cnb_error) = result {
         exit(error_handler.handle_error(lib_cnb_error));
     }
+
+    match read_buildpack_toml::<BM, E>() {
+        Ok(buildpack_toml) => {
+            if buildpack_toml.api != LIBCNB_SUPPORTED_BUILDPACK_API {
+                eprintln!("Error: Cloud Native Buildpack API mismatch");
+                eprintln!(
+                    "This buildpack uses Cloud Native Buildpacks API version {}, but the underlying libcnb.rs library requires CNB API {}.",
```

Comment:
`This buildpack` might not be that helpful if you've got a bunch of buildpacks. Maybe add `buildpack_toml.name` to this message?

Reply:
Good point, I added this in 7cf02ed! 👍🏻